geoh5py/objects/drillhole.py

```diff
@@ -211,7 +211,16 @@ def surveys(self):
         :obj:`numpy.array` of :obj:`float`, shape (3, ): Coordinates of the surveys
         """
         if (getattr(self, "_surveys", None) is None) and self.existing_h5_entity:
-            self._surveys = self.workspace.fetch_coordinates(self.uid, "surveys")
+            surveys = self.workspace.fetch_coordinates(self.uid, "surveys")
+            if len(surveys) == 1:
```

Comment:
**suggestion (test):** have a unit test for survey with a single value

Reply:
Added